File: src/models/body/Asshole.java
package models.body;

import models.items.Item;

/**
 * well, it's an asshole. What do you expect
 */
public class Asshole extends Fuckable {
    public Asshole(Item item, String description) {
        super(item, description);
    }

    public Asshole(Item item, String description, int capacity, int sensivity, int wettness) {
        super(item, description, capacity, sensivity, wettness);
    }
}

File: Assets/Scripts/body/Asshole.cs
using items;

namespace body {
	/**
	 * well, it's an asshole. What do you expect
	 */
	public class Asshole : Fuckable {
		public Asshole(Item item, string description) : base(item, description) {
	        
	    }

	    public Asshole(Item item, string description, int capacity, int sensivity, int wettness) 
			: base(item, description, capacity, sensivity, wettness) {

	    }
	}
}
